docs/docs/gatsby-cli.md

```diff
@@ -70,6 +70,7 @@ At the root of a Gatsby site, compile your application and make it ready for dep
 |       `--prefix-paths`       | Build site with link paths prefixed (set pathPrefix in your config)                                       |
 |        `--no-uglify`         | Build site without uglifying JS bundles (for debugging)                                                   |
 | `--open-tracing-config-file` | Tracer configuration file (OpenTracing compatible). See [Performance Tracing](/docs/performance-tracing/) |
+|         `--noColor`          | Sets the `process.env.FORCE_COLOR` variable to disable colored terminal output)                           |
```

Comment:
```suggestion
|         `--noColor`          | Disables colored terminal output                                                                          |
```

Reply:
Alright, I changed the description to what you suggested.